cmd/kubeadm/test/cmd/config_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeadm
+
+import (
+	"fmt"
+	"math"
+	"strings"
+	"testing"
+)
+
+func TestCmdConfigImagesList(t *testing.T) {
+	var tests = []struct {
+		name     string
+		args     string
+		expected bool
+	}{
+		{
+			name:     "stable Kubernetes version should not throw the warning that a supported etcd version cannot be found",
+			args:     "--kubernetes-version=stable-1",
+			expected: false,
+		},
+		{
+			name:     "invalid(too new) Kubernetes version should throw a warning that the supported etcd version cannot be found",
```

Comment:
```suggestion
			name:     "latest stable Kubernetes version should throw a warning that the supported etcd version cannot be found",
```

one concern is that we may annoy the rest of the project with this in case we forget to push a change in the map. might be a good idea to include code detail? e.g. "The map is called foo and defined in file /path/bar"

wdyt?

Reply:
Just removed this invalid case, I don't think that's what we need to focus on.